README.md

```diff
@@ -24,21 +25,30 @@ ________
 ## Installation
 To install the Scorecards GitHub Action, you need to:
 
-1) Create a Personal Access Token (PAT) for authentication and save the token value as a repository secret; 
+1. (Optional) If you want to use the [Branch-Protection](https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection) check, create a Personal Access Token (PAT) for authentication and save the token value as a repository secret; 
```

Comment:
I prefer the order of having the optional authentication step announced first, but Azeem's question made me realize that there's a few problems with this section. Let's rework it:

We can delete``To install the Scorecards GitHub Action, you need to:`` since there aren't two required steps anymore.

Under installation, let's say:

_____
The Scorecards Action is installed by setting up a workflow on the GitHub UI. [don't link to instructions here]

Note: One Scorecards check (Branch-Protection) requires authentication using a Personal Access Token (PAT).  If you want all Scorecards checks to run, you will need to follow the optional Authentication step. If you don't, all checks will run except Branch-Protection.

Optional Authentication: Create a PAT... [link to instructions]
Required: Set up the workflow via the GitHub UI... [link to instructions]
___

Also: "Supported triggers and repositories" doesn't make much sense where it is, between the two setup steps. Could it come later, either just before the "View Results" section under its own heading, or maybe in a combined new section heading together with "View Results?"

Reply:
> I prefer the order of having the optional authentication step announced first, but Azeem's question made me realize that there's a few problems with this section. Let's rework it:
> 
> We can delete`To install the Scorecards GitHub Action, you need to:` since there aren't two required steps anymore.
> 
> Under installation, let's say:
> 
> The Scorecards Action is installed by setting up a workflow on the GitHub UI. [don't link to instructions here]
> 
> Note: One Scorecards check (Branch-Protection) requires authentication using a Personal Access Token (PAT). If you want all Scorecards checks to run, you will need to follow the optional Authentication step. If you don't, all checks will run except Branch-Protection.
> 
> Optional Authentication: Create a PAT... [link to instructions] Required: Set up the workflow via the GitHub UI... [link to instructions]

changes made

> Also: "Supported triggers and repositories" doesn't make much sense where it is, between the two setup steps. Could it come later, either just before the "View Results" section under its own heading, or maybe in a combined new section heading together with "View Results?"

I initially had in later, but was worried some users may install scorecard and realize too late it does not work for their use case.

Wdut?

Reply:
I think having that section earlier, then. Maybe it's time for a clearer opening/overview paragraph at the top describing Scorecards/Scorecards Action in a sentence or two, and then listing when/who can use the Action. If you agree, I can write something up.

Reply:
Let's not block this merge. Just move the Triggers info (without heading) before the Installation section, and we can rework the first section in another PR.